game/state/city/building.cpp

```diff
@@ -96,6 +97,48 @@ bool Building::hasAliens() const
 	return false;
 }
 
+void Building::initBuilding(GameState &state)
+{
+	// Initialize economy data, done in the map/city editor or when game starts for the first time
+	// Not on save/load, that's why values are serialized
+	currentWage = 65;
```

Comment:
Is this the default initial state? Maybe add it to the extractor instead (IE so it'll be in the 'initial gamestate' - as otherwise this in-code value will override anything set in mods

Reply:
Yes, it was the default internally. I moved it to be a city variable, set in the extractor (only for human map). The rest of "magic numbers" is a rough proportion of this one, so I got rid of those.